sphinxcontrib/images.py

```diff
@@ -343,9 +345,9 @@ def inner_wrapper(writer, node):
     app.env.remote_images = {}
 
 def setup(app):
-    global DEFAULT_CONFIG
     app.require_sphinx('1.0')
-    app.add_config_value('images_config', DEFAULT_CONFIG, 'env')
+    app.add_config_value('images_config', {}, 'env')
```

Comment:
Since the current suggestion completely recreates the config-object (in `check_config(app, config)`) we might as well supply an empty dictionary as default value.

It would be pretty however, wouldn't it, to pass in the defaults here? But maybe this is the origin of the problem/issue. Our multiple configurations options are written in the `images_config` dictionary (as opposed to several individual configuration variables), and somehow someone in the past got the copying/updating of the dictionary wrong...

Reply:
This line is just registering a dictionary with Sphinx so that any user supplied config can land there. In the `check_config` it validates that the config has all the properties that are expected later.